plugin/src/sidebar.ts

```diff
@@ -1,6 +1,6 @@
 import html from "../dist/web/sidebar/index.html?raw";
 
-(globalThis as any).reearth.ui.show(html);
+(globalThis as any).reearth.ui.show(html, { width: 370, extended: true });
```

Comment:
You are setting dimensions here through the API and in the html. Please pick one (I recommend the HTML) and rely on the widget align system for extending.

Reply:
done

Reply:
https://github.com/eukarya-inc/reearth-plateauview/pull/29/commits/326aa3062b26c0e076d4cc56cf8ad8237c0cf1e2